src/axis.js

```diff
@@ -172,3 +171,22 @@ export function axisBottom(scale) {
 export function axisLeft(scale) {
   return axis(left, scale);
 }
+
+function maybeMultilineText(format, orient, x, space) {
+  const repeats = [];
+  const lines = [];
+  return (text) => {
+    text
+    .attr(x, space)
+    .text((d, i) => {
+      const cur = format(d) || "";
+      lines[i] = cur.split("\n");
```

Comment:
We should support \r\n here, too.

```suggestion
      lines[i] = cur.split(/\r\n?|\n/);
```